plugins/woocommerce/client/admin/client/shipping/experimental-shipping-recommendations.tsx

```diff
@@ -43,22 +41,14 @@ const ShippingRecommendations: React.FC = () => {
 			countryCode: getCountryCode(
 				settings.general?.woocommerce_default_country
 			),
-			isJetpackConnected: _isJetpackConnected(),
```

Comment:
The task list component will also prompt the merchant to connect, so there's no reason to hide the banner if we do not have a WPCOM connection.